packages/blast-core/lib/cudos-contract.js

```diff
@@ -19,18 +23,27 @@ module.exports.CudosContract = class CudosContract {
     this.#contractLabel = contractLabel
     this.#signer = signer
     this.#contractAddress = deployedContractAddress
-    this.#wasmPath = path.join(getProjectRootPath(), `artifacts/${contractLabel}.wasm`)
     this.#gasPrice = GasPrice.fromString(getGasPrice())
 
-    if (deployedContractAddress === null && !fs.existsSync(this.#wasmPath)) {
-      throw new BlastError(`Contract with label ${contractLabel} was not found, did you compile it?`)
+    if (deployedContractAddress === null) {
+      this.#wasmPath = path.join(getProjectRootPath(), `artifacts/${contractLabel}.wasm`)
+      if (!fs.existsSync(this.#wasmPath)) {
```

Comment:
why did you change the structure?

Reply:
changed it to avoid getting wasm path in case `getContractFromAddress()` is used to create contract object. In such a case we don't need a local path to a contract with the same name as the already deployed one